src/input/SimModSuite.h

```diff
@@ -527,6 +527,31 @@ class SimModSuite : public MeshInput {
         }
       }
     }
+    if (MeshAtt.lFaceIdNoMesh.size() > 0) {
```

Comment:
You can remove this check, if size is zero you just iterate over an empty list

Reply:
right